app/grants/templates/grants/index.html

```diff
@@ -146,6 +147,7 @@
           {% include 'grants/detail/side-cart.html' %}
         </div>
       </div>
+      {% endcomment %}
 
```

Comment:
There's a missing closing tag here (ln151: `</div>`) - That's what has knocked the footer out :)

Reply:
Ha, yep that fixed it! Good catch 

Dev feature request: If possible I'd love to have a formatter/linter on the HTML files, like we do with JS files. I've noticed missing or extra closing tags multiple times so far